.cirrus.yml

```diff
@@ -15,6 +16,17 @@ test_task:
       env:
         # This should have the same coverage as the previous task, so skip it.
         SKIP_UPLOAD_COVERAGE: 1
+    - name: Test on Linux (Gccgo)
+      container:
+        image: debian:unstable-slim
+      setup_script:
+        - apt-get -y update
+        - apt-get -y install ca-certificates gccgo-go
+      env:
+        # This should have the same coverage as the previous task, so skip it.
+        SKIP_UPLOAD_COVERAGE: 1
+        # Gccgo doesn't support race test
```

Comment:
This needs more detail. I googled "gccgo race detection" and could find very few documents that support your assertion gccgo doesn't support race detection. One of those I did find was this comment in the Go source: https://go.googlesource.com/go/+/master/src/cmd/go/go_test.go#244. Adding the term "debian" resulted in no obvious explanations for why this needs to be disabled.

Reply:
> Adding the term "debian" resulted in no obvious explanations for why this needs to be disabled.

I can't quite parse this; this comment seems fine to me, it's just documenting a fact, which the source you linked also documented.